services/example01/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "private": true,
+  "name": "example01",
+  "version": "0.0.1",
+  "scripts": {
+    "deploy-function":
+      "AWS_ACCESS_KEY_ID=foobar AWS_SECRET_ACCESS_KEY=foobar SLS_DEBUG=* serverless deploy function -f hello",
```

Comment:
These are ok temporarily to test quickly, though it's risky as it's easy to commit secrets. I'd suggest adding a line in the readme about [configuring a profile](https://serverless.com/framework/docs/providers/aws/cli-reference/config-credentials/)

Reply:
This is definitely for testing purpose and we should indeed manage secrets carefully. Since we don't have real code there and we don't use the actual api, it's not harmful, it just serves as an example